CHANGELOG.md

```diff
@@ -13,8 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
 ### Changed
-
+- Update meeting manager to use new event controller API.
+- Update node module to v3 beta.
 ### Removed
+- Remove support for Plan B as well as Safari (and iOS) 12+. The minimum Safari and iOS supported version is now 13. Also clean up all plan-B code path.
```

Comment:
This seems like a copy from js sdk changes. In this context can we just specify the exact property we remove :)

Reply:
I wasn't sure if I could just put the variable, updated :)